expanding CEI-schema for import of Fontenay charters

NTsch opened the pull request:
This makes several additions and changes to the elements in the current schema (e.g. for lacunae, metamarks, etc.), which are used to represent the contents from the Fontenay TEI corpus in CEI. 

Indentations should be better this time, but there's a formatting difference in line length. I can try to adjust this if it causes problems.

StephanMa commented on my/XRX/src/mom/app/cei/xsd/cei.xsd:
```diff
@@ -1244,13 +1119,12 @@
+                <xs:element ref="c"/>
                 <xs:element ref="expan"/>
                 <xs:element ref="hi"/>
-                <xs:element ref="damage"/>
```
Is there a particular reason why those elements are removed? Is ensured that all documents are valid after merging?

StephanMa commented on my/XRX/src/mom/app/cei/xsd/cei.xsd:
```diff
@@ -4175,9 +4056,13 @@
         </xs:annotation>
         <xs:complexType mixed="true">
             <xs:choice minOccurs="0" maxOccurs="unbounded">
-                <xs:element ref="w"/>
```
same here

StephanMa reviewed (commented):
Please have a look

NTsch commented:
Thanks for pointing these out, I'll need to double check.

NTsch commented:
The `w` and `pc` elements I added to `supplied` became unnecessary due to a change in how that element is converted in this corpus, so I removed them.
The other removal was due to elements being part of groups that were already allowed, and therefore became unnecessary, but I changed this to make it clearer in this update.

The formatting is now also the same as in the live version, to make it a bit clearer as well.

StephanMa commented:
lgtm
